puppetserver/docker-entrypoint.d/99-log-config.sh

```diff
@@ -13,7 +13,10 @@ if [ -n "${CERTNAME}" ]; then
   certname=${CERTNAME}.pem
 else
   echo "* CERTNAME: unset"
-  certname=$(cd "${SSLDIR}/certs" && ls *.pem | grep --invert-match ca.pem)
+  certname=$(cd "${SSLDIR}/certs" && find * -type f -name '*.pem' ! -name ca.pem -print0 | xargs -0 ls -1tr | head -n 1)
```

Comment:
can you maybe add a comment that explains what the statement does / what you're looking for? It's a bit hard to read on the first view.

Reply:
will do